test/suites/static_analysis.sh

```diff
@@ -5,6 +5,8 @@ safe_pot_hash() {
 test_static_analysis() {
   (
     set -e
+    # shellcheck disable=SC2039
+    local OUT
```

Comment:
Not very relevant, you're in a subshell

Reply:
@albertodonato did you see this comment?

Reply:
ah sorry, missed it. Fixed now